control_panel_api/views.py

```diff
@@ -112,10 +111,10 @@ class GroupViewSet(viewsets.ModelViewSet):
 class AppViewSet(viewsets.ModelViewSet):
     queryset = App.objects.all()
     serializer_class = AppSerializer
-    filter_backends = (AppFilter,)
+    filter_backends = (DjangoFilterBackend,)
```

Comment:
Should we instead change `settings.DEFAULT_FILTER_BACKENDS` to `DjangoFilterBackend` and explicitly set `filter_backends` to `SuperusersOnlyFilter` on the viewsets that require it?

Reply:
I personally think we shouldn't.

The reason why have the default filter to `SuperusersOnlyFilter` is to be very conservative and have only super users be able to see everything by default. The idea is to require some "effort" if we really want to open an endpoint to everyone (as in this case).

Reply:
👍